testRunner.html

```diff
@@ -4,46 +4,34 @@
 <script src="node_modules/mocha/mocha.js"></script>
 <script src="node_modules/chai/chai.js"></script>
 <script src="bin/traceur.js"></script>
-<script src="test/test-utils.js"></script>
-<script src="test/test-list.js"></script>
 <script>
-
-mocha.setup({ui: 'tdd'});
-
+window.System = new traceur.runtime.BrowserTraceurLoader();
+System.map = System.semverMap(System.version);
 </script>
-<!-- Unit tests -->
-<script src="test/unit/codegeneration/ExplodeExpressionTransformer.js"></script>
-<script src="test/unit/codegeneration/PlaceholderParser.generated.js" type="module"></script>
-<script src="test/unit/codegeneration/SourceMap.js"></script>
-<script src="test/unit/codegeneration/low_level_tests.js"></script>
-<script src="test/unit/codegeneration/writer.js"></script>
-<script src="test/unit/runtime/Loader.js"></script>
-<script src="test/unit/runtime/type-assertions.js"></script>
-<script src="test/unit/semantics/FreeVariableChecker.js"></script>
-<script src="test/unit/semantics/VariableBinder.js"></script>
-<script src="test/unit/syntax/LineNumbers.js"></script>
-<script src="test/unit/syntax/ParseTreeValidator.js"></script>
-<script src="test/unit/syntax/Token.js"></script>
-<script src="test/unit/syntax/parser.js"></script>
-<script src="test/unit/system/Hello.js"></script>
-<script src="test/unit/system/ScriptTypeTextTraceur.js"></script>
-<script src="test/unit/tools/HTMLImportTranscoder/HTMLImportScriptTraceur.js"></script>
-<script src="test/unit/util/ErrorReporter.js"></script>
-<script src="test/unit/util/url.js"></script>
+<script src="test/test-utils.js"></script>
+<script src="test/test-list.js"></script>
 <div id="mocha"></div>
 
-<!-- Feature tests -->
 <script>
-(function() {
+<!-- Unit tests -->
+System.import('./test/unit/unitTestRunner.js').then(function(module) {
+	return module.unitTestRunner.run().then(function() {
+		console.log('unitTestRunner success');
+	}, function(ex) {
+		document.getElementById('mocha').innerText = ex.stack || ex;
+		console.error('unitTestRunner.run FAILED', ex.stack || ex);
+	});
+}).then(function() {
   'use strict';
-  System.import('./src/runtime/webLoader.js', {}).then(function(module) {
+	<!-- Feature tests -->
```

Comment:
It is part of Annex B because browsers support this... it is still bad practice to do it though.